CONTRIBUTING.md

```diff
@@ -167,13 +167,13 @@ any PMC member:
 1. Merge the release proposal PR into the master branch of the repository. There
    should not be any conflicts as the text in the files that have changed should
    only be changed during this release process.
-1. Create a new branch called **"v$major.$minor"** from the **Release SHA**.
 1. Cherry pick the commit in which the release proposal PR was merged, to pick
    up the file changes.
 1. Create a new commit updating [spec.md](spec.md), [profile](profile.md),
```

Comment:
Is there any reason to not do this as part of the release PR?

Reply:
This is removed!